src/utils/helper_functions/calculations.py

```diff
@@ -0,0 +1,31 @@
+"""
+This module contains helper functions for calculations.
+As of now, it only contains one function, find_median.
+"""
+import numpy as np
+import pandas as pd
+
+
+def find_median(df: pd.DataFrame, feature_name: str) -> (float, int):
```

Comment:
Consider using the [typing tuple annotation suggestions found here](https://docs.python.org/3/library/typing.html#annotating-tuples) for the return type hint. Also, consider using a type linter such as [Mypy](https://mypy-lang.org/) (or others) to help check this work as you go.